server/src/test/java/org/uiautomation/ios/e2e/config/AugmenterTest.java

```diff
@@ -40,6 +40,7 @@
   @BeforeClass
   public void startDriver() {
     driver = new RemoteIOSDriver(getRemoteURL(), SampleApps.uiCatalogCap());
+    driver.findElement(By.xpath("//UIAButton[@name='UICatalog']")).click();
```

Comment:
It's unusual for `@BeforeClass` to be interacting with the app like this.  Can you add a comment explaining why this is necessary?  I'm guessing that the UICatalog app now has a button on the front screen where previously it didn't?